docs/source/model_design/risk_factors.rst

```diff
@@ -206,6 +217,77 @@ Effect Modification
 Mediation
 """""""""
 
+Causal Inference in the Global Burden of Disease Study
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Notably, GBD researchers use an evidence scoring system that is based off of a 
+subset of the Bradford Hill Criteria to evaluate the quality of evidence 
+regarding causal relationships between risk-outcome pairs in GBD. 
+Specifically, before computing the relative risks for a GBD risk factor, GBD 
+researchers evaluate the *risk of bias* among individual studies that 
+investigate the relationship between a risk-outcome pair. Then, GBD 
+researchers additionally evaluate the strength (as a direct result of the 
+relative risk curve they compute), consistency (through evluating between 
+study heterogeneity), and dose-response (through the shape of the relative 
+risk curve) for the computed relative risks for a given risk factor. Using 
+these criteria, GBD researchers create a quantitive quality of evidence score 
+for each risk-outcome pair in GBD.
+
+Specifically, GBD researchers evaluate the risk of bias within individual 
```

Comment:
Do you know if they're just documenting this score, perhaps with the hope of being able to use this info later, or if they do anything with it? in other words, if something just barely clears their threshold criteria for addition as a relative risk, but they believe it to have some confounder, do you know if they do anything to account for the confounder that they've documented?

Reply:
> I think we also discussed that if GBD is using some risk exposure A that has confounding variable, or mediator B, and B is not accounted for in their effect size, that they can use this as long as B is also not accounted for in their relative risk. However, in this scenario, the effect size and relrisk may or may not be usable by us.
> 
> I think that should be noted somewhere within the scope of the risk factor page---do you think this would be a good place for that? I think the other reasonable place would be within a vivarium-specific section, noting that that's something to watch out for when pulling GBD RRs and relative risks.
> 
> You, @alibow might've already noted this and included it in one of your sections---I don't remember in which context we discussed this!

Ah, good point! I will definitely add this in here.

Reply:
I wonder

> This is a really clear explanation of their evidence scoring---thank you for writing it!
> 
> I am still unclear (both from your page and from trying previously to dig into this a bit) on if/how they take any of this into account when modeling a risk factor. like if they simply don't use something that has a strong confounder, if they _do_ somehow take some of these relationships into accounts, or if they find studies for their effect sizes/rel risks that are representative for year/locaton/sex/age, and if that's sufficient for their purposes, because all estimates are at the year-loc-age-sex population level.

Hmm, good question. I am a little unclear on this as well, but my understanding/suspicion is that they account for potential biases/confounding in their MR-BRT analyses. Basically, the data extraction template has a bunch of indicator variables for "adjusted_for_sex = yes/no" and then that will be added as a covariate for that study when they are running the meta-regression for the R-O pair, BUT I am not certain that this is what their process is and I will try to investigate! Let me know if you have any thoughts/ideas about this :)

Reply:
The way i understand how GBD select RO pairs is based following criteria:

1. The likely importance of risk factor to a disease.
2. Data and methodology for scaling the risk exposure.
3. epidemiological evidence to support its causal relationship.
4. The effects can be generalized to a general population.

Reply:
I think this would also be important to add, thanks @yongqx! I was focusing just on #3 here (and I guess a little bit on #4), but that would be good to add for context.